xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1257,12 +1252,12 @@ struct AppModel: ModelProtocol {
     }
     
     static func fetchNicknameFromProfileMemo(
-        state: AppModel,
+        state: AppModel, 
         environment: AppEnvironment
     ) -> Update<AppModel> {
         let fx: Fx<AppAction> = Future.detached {
-            let response = try await environment.userProfile.requestOurProfile()
-            if let nickname = response.profile.nickname {
+            let response = try await environment.userProfile.readOurProfile(alias: nil)
```

Comment:
We were asking for **way** more data than we needed before just to get our nickname.